packages/repo-doctor/src/fetch.js

```diff
@@ -0,0 +1,67 @@
+const fetch = require('node-fetch')
```

Comment:
I think it's a bit weird that there is a root file called `fetch`, that has *some* fetching in it, but not all of it (ie. the locales fetching). Not sure what the solution is here, i'm still wrapping my head around the codebase, but just pointing it out fwiw.

Reply:
I moved the utility fetch function from locales.js to fetch.js.